### DISCUSSION TITLE
Filecoin Community Roadmap - 2022 Q2 Project Contributions

### DISCUSSION THREAD

**Discussion by kaitlin-beegle** [filecoin-project/community#484] [General]
There are many exciting enhancements, improvements, and tools that will be introduced by the Filecoin community in 2022. 

In an effort to better communicate about and collaborate on this work, we will be assembling a community roadmap! Help contribute to this effort (and publicize your team's hard work) by contributing your projects! 

## What? 
The [Filecoin Community Roadmap](https://github.com/filecoin-project/community/discussions/456) is an open-source reference document compiled and managed by the Filecoin community. 

**The current roadmap manager is Kaitlin Beegle (kaitlin@fil.org).** 

**The roadmap will be updated on a quarterly basis.**  Prior to the start of each quarter, the roadmap manager will open up and publicize a Call for Contributions in the Filecoin Community Discussion Forum.  Teams will have a designated window of time during which they can submit new milestones and/or update existing milestone timelines.  Last quarter's project submissions can be found [here](https://github.com/filecoin-project/community/discussions/399). 

**This quarter's Call for Contributions period will end on March 31**  If you already submitted a project during a previous contribution period, no need to resubmit! I will be checking in with all existing project teams to ensure that updates, timeline changes, and other items are captured in the updated Roadmap. 

**Project updates and changes will be incorporated into the Roadmap and made available mid-April.** 

## How?
To contribute your project, simply copy+paste the below template into a new comment on this thread.  

```
Project Name:
Project Description (2-3 sentences):
Key Contributors: 
Team/Organization: 
Milestone(s):
Est Milestone Date:
Relevant Links:
```

Worked example:
```
Project Name: Filecoin Virtual Machine (FVM)
Project Description (2-3 sentences): Add user programmable smart contracts ("actors") to Filecoin.
Contributors: @raulk, @stebalien
Team/Organization: Protocol Labs EngRes - FVM
    Milestone 1: Upgrade the Filecoin Mainnet & all Filecoin implementations to validate the chain via the WASM-based Filecoin Virtual Machine, hosting no user-deployed contracts, only the system built-in actors based on the canonical implementation (forked from Rust actors).

    Est Milestone Date: March 2021 for WASM-based FVM deployment + gas schedule rebalancing

    Milestone 2: Add support for user-programmability to the FVM, supporting both native and the foreign EVM actors
    How this improves Filecoin (1-2 sentences): Users can now deploy custom smart contracts, either native or EVM-based, natively on top of the WASM FVM (including creating and deploying new actors like storage provider DeFi loans, auto-renewing storage deals, etc)

    Est Milestone Date: June 2021 for native/foreign actors integration

Relevant Links: https://github.com/filecoin-project/fvm-project, https://www.youtube.com/watch?v=Vw9syGiG31c
```

**A Note on Projects** 
This roadmap is for tracking projects that will open up new Filecoin use cases, network capacity, or tooling. If you believe your project accomplishes this, please submit it! However, projects that refine, incrementally improve, or compete with existing tools/projects built solely on top of Filecoin, should instead be submitted to the [ecosystem project dashboard](https://develop.ecosystem.filecoin.io/?filters=enabled).

The purpose of this roadmap is to chart projects that expand the scope of the Filecoin ecosystem, not those that deepen existing utilities.

**A Note on Milestones** 
The roadmap organizes projects of various types, scopes, and levels of complexity along a single timeline. Although your project may be tracking multiple 'milestones' or development benchmarks, we are focused on plotting high-level milestones that represent community-facing achievements. 

Most projects will record a single milestone; perhaps a launch date, or an alpha release. However, really large projects may record multiple milestones if each milestone represents a significant change in the public-facing functionality of a tool. 

If you're unsure what level of detail to include for your project, feel free to reach out to the roadmap manager.  You are also welcome to include additional project details, but know that not all of this information may be included in the final roadmap. 

*** 
Lastly, be sure to fill out all sections of the template in order to be included in the final roadmap.

Please note that this roadmap should only capture projects that are currently in development. 'Wish list items', or other projects that you'd like to see completed, should be suggested in the [FIPs Discussion Forum](https://github.com/filecoin-project/FIPs/discussions).

**Comment by laurarenpanda** [comment]
Project Name: filplus.info
Project Description: Filplus.info is the first visual dashboard for the Filecoin Plus project, recording relevant real-time data of each DataCap allocation and verified deal transactions on Filecoin through monitoring both on-chain and off-chain information.
Key Contributors: @beeleelee @laurarenpanda
Team/Organization: FileDrive Labs
Milestone(s): Version 3.0.4:
- More statistics and metrics about the Filecoin Plus program
- Information updating about new notaries from the 3-rd round of Notary elections
- Optimize UX design

Est Milestone Date: Middle of April
Relevant Links: https://filplus.info/

**Comment by kaitlin-beegle** [comment]
After a month of stewarding this request to the Filecoin community, this submission window is now closed. 

Teams that are already accounted for on the roadmap will be contacted for updates, and a Q2 updated roadmap will be available before the end of April. Thanks!
